src/main/resources/schema.graphqls

```diff
@@ -1055,6 +1055,8 @@ input UnreleaseLabware {
     barcode: String!
     """The highest section number taken from the block, if it is a block."""
     highestSection: Int
+    """The work number associated with the unrelease of this labware."""
+    workNumber: String
```

Comment:
Katy mentioned this as mandatory field, so do we need a 'String!' for workNumber?